openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -199,7 +204,13 @@ def process(self):
             if renderer.startswith('renderman'):
                 renderer = 'renderman'
 
-            self._set_default_renderer_settings(renderer)
+        if self.instance.data.get("RenderSetupIncludeLights"):
+            from maya.mel import eval
+            eval('optionVar -iv "renderSetup_includeAllLights" 1')
+        if not self.instance.data.get("RenderSetupIncludeLights"):
+            from maya.mel import eval
+            eval('optionVar -iv "renderSetup_includeAllLights" 0')
+        self._set_default_renderer_settings(renderer)
```

Comment:
Be aware that this is **not** a scene-wide setting but a local machine preference. Another user opening the same scene will not have this set correctly.

If we're trying to set Include All Lights on a project wide level I'd really urge and recommend to use the environment variable `MAYA_RENDER_SETUP_INCLUDE_ALL_LIGHTS` instead.

We've had it before and it's super confusing if one opens a scene on another machine and it renders differently due to this option var. So I'd recommend always using the `MAYA_RENDER_SETUP_INCLUDE_ALL_LIGHTS` environment variable to force a preferred setting in the project.

Reply:
Hi! @BigRoy yes thank you, I'm coming from your thread on chaos's website discussing this, made the adjustment because this would make the option local, I'm figuring out what best way to deal with this

Reply:
@antirotor what do you think?

Reply:
There are two points which need to be done.

1. Properly set Deadline submission parameters to pass the correct settings to render job to ensure consistency in rendering, based on the way we decide below
2. Finding the way to control the behavior as Maya stores this setting locally, which can lead to different behavior on different machines.
- the easiest way is to set the ENV variable in Maya Application settings, but this is defining the behavior system-wide
- I'm not sure if there is any mechanism to add per project ENV variable overrides to Maya in OP and/or how difficult it would be to add it. I would personally prefer to do it in this generic way, instead of creating buttons to set every single thing in OP settings.

Example:
![image](https://user-images.githubusercontent.com/352795/165728604-b962ef43-14b6-4435-b9e2-6d9dafc804a6.png)


This way we'd be giving the user freedom to set this and potentially other things on a project level.